### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by dblnz**

### CODE REVIEW

File: docs/how-to-use-flatbuffers.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # How to use FlatBuffers
 
-Flatbuffers is used to serialise and deserialize some data structures.
+Flatbuffers is used to serialize and deserialize some data structures.
```
Comment (simongdavies) [inline docs/how-to-use-flatbuffers.md:3]: I feel like I should object to this as the previous spelling was correct 😄

**Review by simongdavies — APPROVED** [review]

File: docs/how-to-use-flatbuffers.md (hunk 1 above)
Comment (dblnz) [reply docs/how-to-use-flatbuffers.md:3]: well, if the word following was spelled the same, I wouldn't have changed it 😄

**Review by jprendes — APPROVED** [review]
LGTM.
We should add typo checking in CI, recently @andreiltd did that in runwasi.

**Comment by andreiltd** [comment]
> LGTM. We should add typo checking in CI, recently @andreiltd did that in runwasi.

https://github.com/hyperlight-dev/hyperlight/pull/58

**Pull request merged**